### ISSUE TITLE
[Request] an new entity_water.fx shader that mimics the water plane. 

### ISSUE THREAD

**Issue by perelect** [TheGameCreators/GameGuruRepo#508]
I have been trying to rewrite entity_water.fx so that it will mimic the water planes conditions in gg.
I have been trying t do this for a while but i have had no luck doing this.. 
So i was wondering if it could be added as an request for a future update, one day.

What i was thinking was a new entity shader that acts the same as the existing entity_water.fx .
So you can assign it into any entities fpe and it would mimic the water plane current conditions.
So if you use a lua command or the TABTAB menu and adjust the water color, etc the entity shader will adjust to this color as well.

Cheers

**Comment by Belidos3D** [comment]
I was under the impression we already had one, if i remember right Preben added a separate shader when he did all that work on the shaders, that can be added to any entity to make it water.

https://forum.game-guru.com/thread/217943?page=2#msg2579052

**Comment by perelect** [comment]
Yes there is already a entity_water.fx in GG but i was trying to make a new one that would mimic the main water plane in gg, so if you adjusted the water color sliders the new entity shader would adjust as well.
So the effects from water_basic.fx would update a new entity_water.fx as well.
